provider/resource_rediscloud_active_active_subscription.go

```diff
@@ -640,23 +648,28 @@ func buildSubscriptionCreatePlanAADatabases(planMap map[string]interface{}) []*s
 	}
 
 	// create the remaining DBs with all other modules
-	createDatabases = append(createDatabases, createAADatabase(dbName, &idx, localThroughputs, numDatabases, memoryLimitInGB, createModules)...)
+	createDatabases = append(createDatabases, createAADatabase(dbName, &idx, localThroughputs, numDatabases, memoryLimitInGB, datasetSizeInGB, createModules)...)
 
 	return createDatabases
 }
 
 // createDatabase returns a CreateDatabase struct with the given parameters
-func createAADatabase(dbName string, idx *int, localThroughputs []*subscriptions.CreateLocalThroughput, numDatabases int, memoryLimitInGB float64, modules []*subscriptions.CreateModules) []*subscriptions.CreateDatabase {
+func createAADatabase(dbName string, idx *int, localThroughputs []*subscriptions.CreateLocalThroughput, numDatabases int, memoryLimitInGB float64, datasetSizeInGB float64, modules []*subscriptions.CreateModules) []*subscriptions.CreateDatabase {
 	var dbs []*subscriptions.CreateDatabase
 	for i := 0; i < numDatabases; i++ {
 		createDatabase := subscriptions.CreateDatabase{
 			Name:                       redis.String(dbName + strconv.Itoa(*idx)),
 			Protocol:                   redis.String("redis"),
-			MemoryLimitInGB:            redis.Float64(memoryLimitInGB),
 			LocalThroughputMeasurement: localThroughputs,
 			Quantity:                   redis.Int(1),
 			Modules:                    modules,
 		}
+		if datasetSizeInGB != 0 {
```

Comment:
You've done `!= 0` here and `> 0` in pro subscriptions, both are fine but I'd recommend doing the same for both